### ISSUE TITLE
twitter_signin is not working.

### ISSUE THREAD

**Issue by Graham-72** [backdrop-contrib/twitter#12]
The D7 version uses a function `user_external_load` provided by the user module. This function does not exist in Backdrop's user module.

**Comment by klonos** [comment]
Yep, you are correct @Graham-72. I cannot find `user_external_load` anywhere on the api.b.org site. Even if it was removed from core, that should at the very least be documented. This could be because we forked from D8 where things were being removed. Here's links to api.d.org and a contrib D8 module that seems to have been developed as a `user_external_load` replacement.

https://api.drupal.org/api/drupal/modules!user!user.module/function/user_external_load/7.x
https://www.drupal.org/project/externalauth

**Comment by klonos** [comment]
Pinging @quicksketch and @serundeputy for more info.

**Comment by Graham-72** [comment]
Thanks @klonos I am also going to look at how contrib module gauth does this. I have taken a quick look and got a bit of a surprise when I saw that it was a module I ported to Backdrop about 2 years ago 😃

**Comment by deleuzer** [comment]
I was able to get this working by simply changing user_external_load() to user_load(). I don't know if this produces any negative/security issues, but it does allow for twitter sign in.
